karapace/master_coordinator.py

```diff
@@ -57,17 +58,17 @@ def _perform_assignment(self, leader_id, protocol, members):
                 urls[get_identity_url(member_identity["scheme"], member_identity["host"],
                                       member_identity["port"])] = (member_id, member_data)
 
-        lowest_url = sorted(urls)[0]
-        schema_master_id, member_data = urls[lowest_url]
+        master_url = sorted(urls, reverse=self.election_strat.lower() == "highest")[0]
```

Comment:
`self.election_strat.lower() == "highest"`
Is that always False, right? If yes, Why?

Reply:
For now we would like to inject the possibility of configuring the strategy for master election. Since this is passed from the config, in the future new services can have an updated config , that would allow for a different strategy